timelock-agent/src/main/java/com/palantir/atlasdb/timelock/paxos/LeadershipContextFactory.java

```diff
@@ -102,6 +104,18 @@ public Duration leaderPingRate() {
         return runtime().get().pingRate();
     }
 
+    @Override
+    @Value.Derived
+    public Supplier<Duration> greenNodeLeadershipBackoff() {
+        return () -> runtime().get().greenNodeLeadershipBackoff();
```

Comment:
I don't think this would work - if the object returned by runtime().get() changes, then this would not be reflected in `runtime().get()::greenNodeLeadershipBackoff`.

Reply:
ah we should be using a Refreshable then?